### PR TITLE
Export training materials as CSV

### LINKED ISSUE #108: CSV-export
For a potential collaboration with the BINA folks, we need a CSV export, for all training materials. It could be implemented as github action.

### PR DESCRIPTION
**Pull request by haesleinhuepf**
This introduces a new script that exports a CSV file with training data. This CSV file will be expose in the website. This PR closes #108 

image

@SeverusYixin would you mind reviewing this PR for me? Is the code readable? Does it make sense? Thanks!

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
The github workflow created the CSV file as reported [here](https://github.com/NFDI4BIOIMAGE/training/actions/runs/9869750128/job/27254082057?pr=110#step:5:2851)

File: scripts/export_csv.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+# filter type by by
+def contains_filter_word(text, words):
+    return any(word in text for word in words)
+df = df[df['type'].apply(lambda x: contains_filter_word(x, filter_types))]
+
+# select columns
+df = df[selected_columns]
+df.head(5)
```
Comment (SeverusYixin) [inline scripts/export_csv.py:36]:
```suggestion
```
shall we remove this debug statement?

**Comment by SeverusYixin** [comment]
Do i need to merge pull request?

**Comment by haesleinhuepf** [comment]
> Do i need to merge pull request?

I'm happy to merge it :-)

**Pull request merged**